cyberInt-argos/README.md

```diff
@@ -0,0 +1,7 @@
+# CyberInt Parameters Configuration
+Parameter                           | Name | Default Value | Type | Required (True/False) | Description
+---                                 | --- | --- | --- |--- |---
+hostname                            | Host Name | https://<instance name>.cyberint.io | String | True | IP or URL for the instance.
```

Comment:
@Benimanela that possible to add any URL ? Can we describe it better + add more info about configuring the Cyberint side? let's use IBM & Lior readme as best practices for all the workflows.

Reply:
done